packages/whats-new/src/whats-new-page.js

```diff
@@ -27,9 +28,9 @@ function WhatsNewPage( {
 	return (
 		<GuidePage className="whats-new-page__container">
 			<div className="whats-new-page__text">
-				<h1 className="whats-new-page__heading">{ heading }</h1>
+				{ heading && <h1 className="whats-new-page__heading">{ heading }</h1> }
```

Comment:
Nit: Can this use `@wordpress/components` `Text` with variant set to `title.large`?

Reply:
Note: Text is still experimental so we decided against using it for now.

Reply:
Note: Text is still experimental so we decided to not use it for now.